web/controllers/homepage_controller.ex

```diff
@@ -4,6 +4,9 @@ defmodule App.HomepageController do
   alias App.{CMSRepo, Resources}
 
   @http Application.get_env(:app, :http)
+  @google_sheet_url_email Application.get_env(:app, :google_sheet_url_email)
+  @google_sheet_url_suggestion Application.get_env(
+    :app, :google_sheet_url_suggestion)
```

Comment:
I'm not sure about the indentation. Is it to not have a too long line, should the closing bracket be on a new line?